docs/HyperSync/overview.md

```diff
@@ -28,19 +28,37 @@ E.g. In 10 seconds HyperSync can:
 
 ```python
 import hypersync
-from hypersync import LogSelection, LogField, DataType, FieldSelection, ColumnMapping, TransactionField
+from hypersync import (
+    LogSelection,
+    LogField,
+    DataType,
+    FieldSelection,
+    ColumnMapping,
+    TransactionField,
+)
 import asyncio
 
+
 async def collect_events():
     # choose network
-    client = hypersync.HypersyncClient("https://arbitrum.hypersync.xyz")
+    client = hypersync.HypersyncClient(
+        hypersync.ClientConfig(url="https://arbitrum.hypersync.xyz")
+    )
```

Comment:
As we want to promote token usage we should add a comment at least about a bearer token

Reply:
added